### PR TITLE
Update stacks/java-websphereliberty-gradle/devfile.yaml

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?:

- update schema to `2.2.0`;
- use `https` protocol for the `java-websphereliberty-gradle` devfile.

### Which issue(s) this PR fixes:
_Link to github issue(s)_

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

https://workspaces.openshift.com/#https://raw.githubusercontent.com/akurinnoy/registry/patch-2/stacks/java-websphereliberty-gradle/devfile.yaml

### CODE REVIEW

**Review by thepetk — CHANGES_REQUESTED** [review]
@akurinnoy I think we should also bump up the version of the devfile

https://github.com/devfile/registry/blob/b416798748d457bb2cb3eab9c0305de1492a9c0b/stacks/java-websphereliberty-gradle/devfile.yaml#L34

I'd suggest bumping up to `0.5.0`

**Comment by akurinnoy** [comment]
@thepetk thanks for the feedback, fixed.

**Review by thepetk — COMMENTED** [review]
@ajm01 as you are listed in the OWNERS file could you please review this PR?

**Review by ibuziuk — APPROVED** [review]

**Comment by thepetk** [comment]
> @ajm01 as you are listed in the OWNERS file could you please review this PR?

@ajm01 kind reminder to review the PR as you are listed as an owner. As discussed in the devfiles community call, we will wait for a week, until 23 Apr 2024, so the owner has time to review. In case we don't have a review from the owner, the devfile services team will review & merge the updates to unblock the `eclipse che` team.

**Review by thepetk — APPROVED** [review]
/lgtm

**Pull request merged**
